eclipse-collections/src/main/java/org/eclipse/collections/impl/collection/mutable/AbstractMultiReaderMutableCollection.java

```diff
@@ -1981,13 +1982,11 @@ public void appendString(Appendable appendable, String start, String separator,
 
     public static class ReadWriteLockWrapper
     {
-        private final ReadWriteLock readWriteLock;
```

Comment:
Is readWriteLock is not used?

Reply:
Correct, the field is not used. The parameter is still used in the constructor.

```java
    public static class ReadWriteLockWrapper
    {
        private final ReadWriteLock readWriteLock;
        private final LockWrapper readLock;
        private final LockWrapper writeLock;

        public ReadWriteLockWrapper(ReadWriteLock readWriteLock)
        {
            this.readWriteLock = readWriteLock;
            this.readLock = new LockWrapper(readWriteLock.readLock());
            this.writeLock = new LockWrapper(readWriteLock.writeLock());
        }
...
```

Reply:
Ok